### PR TITLE
testing/code-server: new aport

### PR DESCRIPTION
**Pull request by wenerme**

### CODE REVIEW

**Comment by maxice8** [comment]
please use our commit naming scheme

```
<repo>/<package>: new aport

$url
$pkgdesc
```

**Comment by maxice8** [comment]
Also check drone-ci and fix the failures pointed out

File: testing/code-server/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Maintainer: wener <wenermail@gmail.com>
+pkgname="code-server"
+pkgver="1.1156"
+_fullver="1.1156-vsc1.33.1"
+pkgrel=0
+pkgdesc="Run VS Code on a remote server"
+url="https://github.com/cdr/code-server"
+arch="all"
+license="MIT"
+depends=""
```
Comment (maxice8) [inline testing/code-server/APKBUILD]: empty variables can be removed for brevity

File: testing/code-server/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+makedepends="yarn npm python2"
```
Comment (maxice8) [inline testing/code-server/APKBUILD]: Please use python3 as python2 will soon be EOL

File: testing/code-server/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+subpackages=""
+source="$pkgname-$_fullver.tar.gz::https://github.com/cdr/code-server/archive/$_fullver.tar.gz"
+builddir="$srcdir/$pkgname-$_fullver"
+options="!check"
```
Comment (maxice8) [inline testing/code-server/APKBUILD]: Please add a comment on why check is disabled

**Comment by wenerme** [comment]
py3 will failed node-gyp, newer node-gyp need upstream upgrade https://github.com/nodejs/node-gyp/issues/1687 node-gyp support py3 lately.

**Pull request closed without merging**